yw_website/yw_website/apps/website/views.py

```diff
@@ -123,12 +127,13 @@ def detailed_workflow(request, workflow_id, version_id):
                 "tags":tags,
                 "run_list": runs,
                 "form": form,
+                "edit": edit,
             }
             return render(request, "pages/detailed_workflow.html", info)
         elif request.method == "POST":
             new_version = request.POST["version_id"]
             return redirect(
-                "/detailed_workflow/{}/version/{}/".format(workflow_id, new_version)
+                "/detailed_workflow/{}/version/{}".format(workflow_id, new_version)
```

Comment:
you need to put the `/` back at the end of this route. This will cause a `404` error if we try to redirect to this.